_includes/tutorials/kafka-connect-datagen-ccloud/kafka/markup/dev/check-connector.adoc

```diff
@@ -1,11 +1,23 @@
-Check the status of the connector:
+To check the status of the connector from the command line, you have the same two options as provisioning.
+
+*Option 1.* Using the `ccloud` CLI.
 
 +++++
 <pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-connect-datagen-ccloud/kafka/code/tutorial-steps/dev/check-connector.sh %}</code></pre>
 +++++
 
-After a minute or two, rerun this command to get the updated Status, it will change from `PROVISIONING` to `RUNNING` when the Connector is ready.
+Rerun this command to get the updated Status, it will change from `PROVISIONING` to `RUNNING` when the Connector is ready.
 
 +++++
 <pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-connect-datagen-ccloud/kafka/code/tutorial-steps/dev/check-connector_expected.log %}</code></pre>
 +++++
+
+*Option 2.* The Confluent Cloud REST API provides a `connector_name/status` endpoint you can use to verify the status of a provisioned connector. Note the `connector.state` field in the returned JSON.
```

Comment:
I can see people trying to run this step without having create the necessary api-key/secret in the step previously. If you feel similarly, can you add something here that mentions "An API Key and Secret which authorizes us to control our cloud account is needed to complete this command, see Option 2 in the previous step for instructions on how to create it". (I'm no wordsmith :) )